jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/ContextHandler.java

```diff
@@ -1102,15 +1102,17 @@ public void setDisplayName(String displayName)
     /**
      * @return Returns the resourceBase.
      */
+    @ManagedAttribute("document root for context")
     public Resource getBaseResource()
     {
         return _coreContextHandler.getBaseResource();
     }
 
     /**
      * @return Returns the base resource as a string.
+     * @deprecated use #getBaseResource()
      */
-    @ManagedAttribute("document root for context")
+    @Deprecated
```

Comment:
Can't we just delete this method now?

Reply:
I think we should keep it in ee9 as we are trying to be as compatible as possible with jetty-10.
We can remove in 12.1.0

Reply:
We already changed the contract from jetty-10 though.

`Resource.newResource(String)` now doesn't exist.
Have to use `ResourceFactory.newResource(String)` now.

Reply:
yeah, there are changes, but it costs us nothing much to hold onto a deprecated method for at least a few release cycles.
We could remove it, but it can be a confusing name change, so leaving it as a sign post is a good idea.